src/TorchSharp/TorchSharp.csproj

```diff
@@ -29,6 +29,7 @@
 
   <ItemGroup>
     <PackageReference Include="Google.Protobuf" Version="3.21.9" />
+    <PackageReference Include="Razorvine.Pickle" Version="1.4.0" />
```

Comment:
If we're introducing a new dependency, we're going to have to take a look at its license, etc.

Reply:
Definitely not -- it's not nice to the author of the package, who deserves to have dependents noted, and it's essentially means forking, which has software engineering implications.

Reply:
Agreed :)